stellar/utils.ts

```diff
@@ -39,6 +38,18 @@ const SUPPORTED_CONTRACTS = new Set([
     'Multicall',
 ]);
 
+type NetworkType = 'local' | 'futurenet' | 'testnet' | 'mainnet';
+
+interface Options {
```

Comment:
Use a different name maybe? These options are different from the CLI provided options so a bit confusing